arxivdigest/frontend/templates/signup.html

```diff
@@ -1,37 +1,33 @@
 {% extends "base.html" %} {% block content %}
     <div class="container">
+        <h2>Sign up</h2>
+        <div class="bottom_border">
+            <p>Fill out this form to register yourself. The fields marked with * are mandatory.</p>
+        </div>
         <fieldset>
-            <legend>Signup:</legend>
-
             <form action="{{ url_for("general.signup") }}" method="POST" id="signupForm" enctype=multipart/form-data>
                 <table>
                     {% include 'user_profile_form.html' %}
                     <tr>
                         <td>
                             <label for="termsandconditions" class="signupLabel">Terms and Conditions: *</label>
                         </td>
-                    <td>
-                        <br>
-                        <p>I have read and accepted the terms and conditions
-                            <a href="{{ url_for("general.termsandconditions") }}">here</a></p>
-                        <input type="checkbox" name="termsandconditions" value="accepted" required id="tc_checkbox">
-                    </td>
-                    <td></td>
+                        <td><input type="checkbox" name="termsandconditions" value="accepted" required id="tc_checkbox">
+                             I have read and accepted the <a href="{{ url_for("general.termsandconditions") }}">Terms and conditions</a>
```

Comment:
Broken link because of rename:
Could not build url for endpoint 'general.termsandconditions'. Did you mean 'general.terms_and_conditions'

Reply:
Also the formatting here might not be as intended:
![image](https://user-images.githubusercontent.com/2963225/77169552-f7605100-6ab9-11ea-8909-74f54e5e76f6.png)

Reply:
Fixed